firestore/incidents.py

```diff
@@ -280,17 +278,24 @@ def get_user_report_by_report_id(report_id):
 
         # Update the document with the new details
         updates = {}
+        # User updates fields
         if user_report.get("contact_name"):
             updates['contact_name'] = user_report["contact_name"]
         if user_report.get("email"):
             updates['email'] = user_report["email"]
         if user_report.get("phone"):
             updates['phone'] = user_report["phone"]
-        if user_report.get("status"):
-            updates['status'] = user_report["status"]
+            
```

Comment:
I think we should delete the current updateUserReport(). Rename the insertUserReport to updateUserReport. 

The current updateUserReport is not using the ORM lib, therefore is not protecting our data properly.